drivers/gpu/drm/panel/panel-waveshare-dsi-v2.c

```diff
@@ -0,0 +1,1353 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright © 2023 Raspberry Pi Ltd
```

Comment:
More copy pasta.

Reply:
Okay, this is a copy. I'll fix this.
Any other suggestions? I'll standardize the changes.

Reply:
Previously mentioned issues have been fixed. PTAL.